lib/metric_fu/metrics/reek/generator.rb

```diff
@@ -27,7 +27,7 @@ def run!(args)
     end
 
     def analyze
-      @matches = @output.chomp.split("\n\n").map { |m| m.split("\n") }
+      @matches = @output.chomp.split("\n\n").map { |m| m.gsub(/\n\s+\[/, "SPLIT_ME_HERE_PLEASE[").split("SPLIT_ME_HERE_PLEASE") }
```

Comment:
IMHO the code would look better, but the regex will indeed be less obvious. @bf4, thoughts?